com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Parser/HrJobRoles.php

```diff
@@ -418,26 +418,20 @@ function fini() {}
 
 
   /**
-   * get the option database ID given its label or ID
+   * get the Option Value database value given its label
```

Comment:
Please make the first letter uppercase and add a space between the method description and the @params block